### DISCUSSION TITLE
Timber connections for a pin jointed tower.

### DISCUSSION THREAD

**Discussion by domacas** [idea-statica/ideastatica-public#409] [Rhino/Grasshopper]
Hello,

My name is Dom and I am a final year MSc Civil Engineering student from TU Delft. My graduation thesis concerns the design and optimization of a hyperboloid lattice timber tower following different topologies (please see attached image). However, a more detailed design stage requires one to look into connections that might entail steel - timber ones. As my current workflow involves an extensive use of Grasshopper and Karamba3D plug-in I wondered if the KarambaIDEA plug-in would be able to consider timber members being joining in one node for the design of a joint? I am asking because the plug-in examples mainly considered steel as the main material. I was wondering if the material (such as GL24H) can be accepted in providing designs for such an assembly? Thank you for your help and consideration!

Kind regards,
Dom Acas
![IDEA_2](https://github.com/idea-statica/ideastatica-public/assets/108278551/c735ae47-5f19-4254-a076-12e1bac35aed)

![IDEA_1](https://github.com/idea-statica/ideastatica-public/assets/108278551/c853473c-25fe-4c8f-ae62-a9abdf416050)

**Comment by Sebastiaanvhassel** [comment]
Dear Dom Acas, 

It's great to hear that you are considering the use of IDEA StatiCa for your graduation thesis. 

Unfortunately, we do not have Timber material available in our IDEA Open Model (IOM) at the moment. Therefore, it is not available in KarambaIDEA or our IDEA Grasshopper plugin (https://www.ideastatica.com/nl/support-center/idea-grasshopper-plugin).
Additionally, the special manufacturing operations related to timber, such as gusset plates, cannot be applied in IOM. 

However, you could model the elements using steel sections and then change the general material properties to those of GL24h. In this way, the general manufacturing operations such as stiffening plates and bolt grids can be applied. Finally, you can change the steel sections into timber once exported. 
Moreover, IDEA StatiCa steel-to-timber joints are there at the moment only for the check of steel plates and determination of force vectors in fasteners. Code checks of timber members, bolts, and dowels are not delivered and must be performed separately. On the other hand, IDEA StatiCa Connection application delivers acting shear and tension forces on each bolt or dowel for the precise manual code-check. 

Alternatively, you could export the internal forces and design your connection separately using IDEA StatiCa Connection application. For inspiration, consider watching this webinar: 
https://www.ideastatica.com/webinars/connection-wednesdays-optimization-of-timber-column-anchoring

May take a look at one of these articles: 
https://www.ideastatica.com/blog/designing-steel-to-timber-connections
https://www.ideastatica.com/support-center/timber-connections-steel-to-timber
https://www.ideastatica.com/support-center/steel-to-timber-joints-theoretical-background

We also recommend exploring the available examples to get started with generating the connections in IOM (using Grasshopper plugin).  https://developer.ideastatica.com/docs/extensions/grasshopper/grasshopper_examples.html 
The examples can be found in the installation folder. You could start with “IOM_Create IOM connection with IOM and Results”. 

Best of luck with the rest of your thesis. Feel free to contact us if you have any further questions. 

Kind regards,  

IDEA StatiCa Support Center
